app/routes/rentals_routes.py

```diff
@@ -0,0 +1,68 @@
+from app import db
+from flask import Blueprint, jsonify, make_response, request
+from ..models.rental import Rental
+from ..models.customer import Customer
+from ..models.video import Video
+from app.models.validation import validate_model, validate_request
+
+# ~~~~~~ initialize rentals blueprint ~~~~~~
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def checkout_video():
+    reqs = {"customer_id", "video_id"}
+    request_body = validate_request(request, reqs)
+
+    video_id = request_body["video_id"]
+    video = validate_model(Video, video_id)
+
+    customer_id = request_body["customer_id"]
+    customer = validate_model(Customer, customer_id)
+
+    if video and customer:
+        if video.available_inventory<=0:
+            return make_response(jsonify({"message": f"Could not perform checkout"}), 400)
+
+        new_rental = Rental.from_dict(request_body)
+        
+        new_rental.customer = customer
+        new_rental.customer.videos_checked_out_count += 1
```

Comment:
Great example making use of the relationships you've set up, to be able to cleanly use SQLAlchemy's OOP interface!